spec/factorial.js

```diff
@@ -6,16 +6,16 @@ import factorial from '../src/factorial'
 // multiple test cases
 // reference to chai library
 
-describe('factorial', function(){
-  context('when given a number', function(){
-    it('should retrn factorial', function() {
-      expect(factorial(5)).to.be(120)
+describe('factorial', () => {
+  context('when given a number', () => {
+    it('should return the factorial', () => {
+      expect(factorial(5)).to.equal(120)
     })
-  })
+  });
 
```

Comment:
Add test for `factorial(0)` as well to ensure you have the correct base case in your recursive function.